advent-of-code/rucksack-problem/README.md

```diff
@@ -0,0 +1,44 @@
+# rucksack-problem
```

Comment:
Fill in some short instructions here, or remove them if not relevant.

Reply:
Yes, I added a brief description.